nodeclient/http_api_client.go

```diff
@@ -391,7 +397,9 @@ func (client *Client) OutputByID(ctx context.Context, outputID iotago.OutputID)
 	query := fmt.Sprintf(RouteOutput, outputID.ToHex())
 
 	res := &OutputResponse{}
-	if _, err := client.Do(ctx, http.MethodGet, query, nil, res); err != nil {
+	if _, err := client.DoWithRequestHeaderHook(ctx, http.MethodGet, query, func(header http.Header) {
+		header.Set("Accept", "application/json")
```

Comment:
define and use `MIMEApplicationJSON = "application/json"` since you are also mentioning it in the comments with that name